types/ember-testing-helpers/tsconfig.json

```diff
@@ -14,7 +14,8 @@
         ],
         "types": [],
         "noEmit": true,
-        "forceConsistentCasingInFileNames": true
+        "forceConsistentCasingInFileNames": true,
+        "allowSyntheticDefaultImports": true
```

Comment:
Correction, I *did* already update and push this change; GitHub isn't showing it for some reason. Going to try to forcibly update it with a changed commit message or something.